modules/_labs/color-picker/react/lib/ColorPicker.tsx

```diff
@@ -0,0 +1,174 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {ColorInput} from '@workday/canvas-kit-react-color-picker';
+import {borderRadius, colors, spacing} from '@workday/canvas-kit-react-core';
+import {Label} from '@workday/canvas-kit-react-form-field';
+import {IconButton} from '@workday/canvas-kit-react-button';
+import {checkIcon} from '@workday/canvas-system-icons-web';
+
+import {SwatchBook} from './parts/SwatchBook';
+import {ResetButton} from './parts/ColorReset';
+
+export interface ColorPickerProps {
+  ariaLabel?: string;
+
+  onChange: (color: string) => void;
+  selectedColor?: string;
+  customColors?: string[];
+
+  showCustomInput?: boolean;
+  customInputLabel?: string;
+  onSubmitClick?: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
+
+  onReset?: Function;
+  resetColor?: string;
+  resetLabel?: string;
```

Comment:
We're in a weird transition right now. We use the `README.md` file to document props, but we now have support in Storybook to document props using JSDoc inside the prop interface. We will eventually deprecate documenting props in the README (duplicated and harder to maintain). In the meantime, we've added prop documentation to _both_

Reply:
so, I should JSDoc these?

Reply:
We are doing this to all other components, so that would help us a lot with that effort.